core/src/test/java/com/linecorp/armeria/client/logging/LoggingClientTest.java

```diff
@@ -152,4 +154,28 @@ void sanitizeRequestContent() throws Exception {
         // Ensure sanitize the request content of the phone number 333-490-4499
         assertThat(ctx.logBuilder().toString()).doesNotContain("333-490-4499");
     }
+
+    @Test
+    void exceptionWhileLogging() throws Exception {
+        final HttpRequest req = HttpRequest.of(RequestHeaders.of(HttpMethod.POST, "/hello/trustin",
+                                                                 HttpHeaderNames.SCHEME, "http",
+                                                                 HttpHeaderNames.AUTHORITY, "test.com"));
+        final ClientRequestContext ctx = ClientRequestContext.of(req);
+        final Logger logger = LoggingTestUtil.newMockLogger(ctx, capturedCause);
+        final LoggingClient loggingClient = LoggingClient.builder()
+                                                         .logger(logger)
+                                                         .requestLogLevelMapper(log -> {
+                                                             throw new AnticipatedException();
+                                                         })
+                                                         .responseLogLevelMapper(log -> {
+                                                             throw new AnticipatedException();
+                                                         })
+                                                         .build(delegate);
+        loggingClient.execute(ctx, req);
+        verify(logger).warn(eq("Unexpected exception while logging request: "), any(CompletionException.class));
```

Comment:
```suggestion
        verify(logger).warn(contains("Unexpected exception while logging request: "), any(CompletionException.class));
```

This test was failed https://ci.appveyor.com/project/line/armeria/builds/35079753/job/5fn7wowbu1vdvwwf#L1498 😉

Reply:
Oops forgot to fix the test. 🤣